### ISSUE TITLE
HW-"In your first ggplot command, you didn't get the data layer right."

### ISSUE THREAD

**Issue by bonnyp** [daroczig/CEU-R-lab#16]
In Week3 homework, on the "A Boxplot" question, I am fairly certain my code is correct in providing the number of diamonds based on cut category in a barplot.  But my error msg says "In your first ggplot command, you didn't get the data layer right."  

I followed example on Week 3 (https://github.com/daroczig/CEU-R-lab/blob/2018/3.R#L129) and checked several online sites for syntax.  I even see the barplot generated ok.

Without showing my code to everyone, is anyone else stuck on this?

**Comment by daroczig** [comment]
Just to make sure it's not a wording issue in the exercise, this is the output I'd like you to generate with a `ggplot` call:

![image](https://user-images.githubusercontent.com/495736/31254555-6f9efa8a-aa29-11e7-986a-193ef8a74b69.png)

Does your plot looks like this one? (Except for the colors, sorry, was using a custom theme here)

PS the exercise says:

> Use the `ggplot2` package to generate a boxplot on the prices split by the `cut` quality.

So it's not the number of diamonds per `cut`, but the distribution of the `price` per `cut`

Let me know if this resolves the issue and we can close the ticket

**Comment by AJXander** [comment]
I can report the same issue. My boxplot resembles the example above:
![image](https://user-images.githubusercontent.com/32059162/31254670-fa547d30-aa29-11e7-8a4e-34369d0047d0.png)
But I get the same error, that my ggplot data layer is incorrect.

**Comment by daroczig** [comment]
That plot looks good. Any chance that you rendered multiple plots in the same submission? Eg having multiple `ggplot` calls and only the last once being the correct one? As I understand, datacamp currently checks the first plot after hitting the "submit answer" button.

To make sure you get that, can you please copy your code generating the plot (should be 1 line), refresh the page, paste the `ggplot` command and hit "submit answer". Does it resolve the problem?

**Comment by AJXander** [comment]
I have only one line of code; if you'll excuse me posting it in full you will see it in the attached screenshot.

I have refreshed the page  (Ctrl + F5), then refreshed to the default code and submitted the answer again; it still fails. The same issue affects every ggplot task throughout the exercise. I can create the requested plots, but the data layer is listed as incorrect.

EDIT: Removed screenshot of different exercise indicating the same error.
